### ISSUE TITLE
Sticky footers don't work in this theme

### ISSUE THREAD

**Issue by bhartvigsen** [braginteractive/MDLWP#42]
Hi!

I've spent all morning and most of the afternoon trying to get sticky footers to work in this theme. All of the usual tricks (relative container, absolute footer: https://getbootstrap.com/examples/sticky-footer) DO NOT WORK! I had kind of bastardized your theme a bit and was blaming myself for breaking it somehow, but I've gone back and re-cloned the repo and I can replicate the issue on an untouched theme (it looks sticky and mostly is, but try making an empty page and look at it with a tall browser window). I've found that if I remove the material CSS file (the big one) the problem goes away but I've been totally unable to determine the offending line(s) as divide-and-conquer is failing me. All I could figure out is that the problem is somewhere between 0-75% of the file. For some reason neither removing the top or bottom half of the CSS eliminates the problem.

I doubt that the material CSS is _actually_ faulty, I think it's more likely that something in this theme is causing a conflict with it, but I've so far been unable to determine the problem, so I come to the issues page, hat in hand, begging for some insight on why a normal sticky footer technique fails to work with this theme. I'm really happy with *everything else* in this theme but I'm completely stuck on this one issue.

**Comment by braginteractive** [comment]
This theme is based off of Material Design Lite https://getmdl.io/components/index.html, not Bootstrap. So I wouldn't expect the Bootstrap markup to work with the theme. 

You might check out how MDL does the fixed headers: https://getmdl.io/components/index.html#layout-section/layout 

Then create your custom styles for a fixed footer. FYI, MDL is based on Flexbox so the ways you are trying to do it with the Bootstrap examples are not going to work. 

This is a customization to the theme you are trying to do, don't think I would consider it an issue. Everyone has different edits/customizations they want to do to themes. I simply can't support every variation.

**Comment by bhartvigsen** [comment]
I'm not interested in a fixed footer. I want a sticky footer. Also the footer in your base theme doesn't even work properly on small pages, hence the reason I opened an issue. I wasn't using bootstrap markup, I was using basic CSS. There is an established method for using sticky footers and I just linked to an example showing it.

If you have no insight on why sticky footers don't work in this theme I'm going to have to abandon my work on it and use something else. It's unfortunate but at least I'm figuring this out two days into my project and not two weeks. Regardless, thanks for your efforts in this theme.
